.gitignore

```diff
@@ -0,0 +1 @@
+__ pycache__
```

Comment:
Typo? should be `__pycache__` without a space before p

Reply:
Title could be `Add __pycache__ to gitignore`

Reply:
ps. you cannot change the title now, that would need a history rewrite of git, which we do not want... this is just a hint for future commits